backend/server.ts

```diff
@@ -1,6 +1,7 @@
 import express, { Request, Response, NextFunction } from "express";
 import User from "./models/user";
 const app = express();
+import Gardens from './models/garden';
```

Comment:
Change Gardens to Garden, this'll help keep our models consistent throughout our project